docs/docker.md

```diff
@@ -88,3 +88,53 @@ docker run -d -p 3013:3013 --hostname node0 \
 ```
 
 **Note: make sure `hostname` option is set when reusing the mnesia data directory**
+
+# Localnet
+
+Small local network (not connected to testnet) can be created with `docker-compose`.
+It runs three nodes using the `mean16s-generic` miner (fastest generic miner) and a proxy server to allow CORS.
+
+Both external and internal API are exposed to the docker host, the URL pattern is as follows:
```

Comment:
How is user meant to reach WebSocket internal endpoint (3114) of each node? (See [overview of endpoints](https://github.com/aeternity/protocol/blob/0b4b55f3a533770c82758be7b01942d5b74e8344/epoch/api/README.md#overview).)

Reply:
See https://www.pivotaltracker.com/story/show/154904411